pkg/discord/main.go

```diff
@@ -425,13 +488,49 @@ func leaveProject(data CommandData) {
 	}
 }
 
+// Set project champion(s)
+func setChampions(data CommandData) {
+	projectName := data.Args[0]
+	users := data.Args[1:]
+	for _, user := range users {
+		userID := strings.TrimSuffix(strings.TrimPrefix(user, "<@"), ">")
+		if discordUser, err := data.Session.User(userID); err != nil {
+			if _, err := data.Session.ChannelMessageSend(data.ChannelID, "Failed to find user " + user); err != nil {
+				fmt.Println("error sending failed to find user message,", err)
+			}
+		} else {
```

Comment:
i wonder if we can make this a bit less nested by returning from the previous if statement, something like
```
if truty {
  ...
  return 
}
// continue with falsey case here
```

Reply:
We can, but we need access to the `discordUser` variable so it would have to be:
```
discordUser, err := expr()
if err != nil {
return
}
// falsey case
```

Reply:
gotcha, yeah i think it would read better to not be so nested. Lmk what you think.. small potatoes - otherwise it looks good